src/Lavary/Menu/Builder.php

```diff
@@ -758,6 +758,24 @@ public function conf($key)
         return $this->conf[$key];
     }
 
+     /**
+     * Set/merge custom configuration data
+     *
+     * @param array $config
+     * @param bool $merge
+     * @param string $configGroup
+     *
+     * @return string
+     */
+    public function config(array $config, bool $merge = true, string $configGroup = 'default')
+    {
+        if($merge){
```

Comment:
`if($merge){` -- Could you please follow the syntax of nearby code. So, to match the function above, this should be `if ($merge) {`